subiquity/models/subiquity.py

```diff
@@ -81,6 +82,7 @@ def setup_yaml():
     "packages",
     "snaplist",
     "ssh",
+    # "updates", # FIXME hey mwhudson - do I need this?
```

Comment:
Not unless you want to force the client to provide a value for this -- which I don't think we do.

Reply:
Agree, removing this line